doajtest/mocks/bll_article.py

```diff
@@ -26,7 +26,7 @@ def discover_duplicates(cls, doi_duplicates=0, fulltext_duplicates=0, overlap=0)
 
         possible_duplicates = {"doi" : [], "fulltext" : []}
         for i, ident in enumerate(idents):
-            source = ArticleFixtureFactory.make_article_source(eissn="1234-5678", pissn="9876-5432", doi=ident["doi"], fulltext=["fulltext"])
+            source = DoajXmlArticleFixtureFactory.make_article_source(eissn="1234-5678", pissn="9876-5432", doi=ident["doi"], fulltext=["fulltext"])
```

Comment:
Another example, `make_article_source` returns the core article format as a python dictionary, rather than XML.